chat/message-rating/src/MessageRating.spec.tsx

```diff
@@ -7,92 +11,144 @@ import { MessageRatingProps } from './MessageRating/MessageRating.types';
 const onChange = jest.fn();
```

Comment:
Even though everything is passing as expected in this file, I don't think we should globally define `onChange`. It could lead to some unexpected results because the state might persist across all tests. I remember running into issues with this with the `Select` component. I would recommend defining this in individual tests.